Scripts/NodeDataCache.cs

```diff
@@ -200,12 +192,17 @@ private class PortDataCache : Dictionary<System.Type, List<NodePort>>, ISerializ
             public void OnAfterDeserialize() {
                 this.Clear();
 
-                if (keys.Count != values.Count)
-                    throw new System.Exception(string.Format("there are {0} keys and {1} values after deserialization. Make sure that both key and value types are serializable."));
```

Comment:
This was the only bug fixed in the PR; missing arguments supplied to `string.format` for the keys, values count.